front/.env.local

```diff
@@ -1 +1,2 @@
 VUE_APP_EXTENSION_BASE_URL=http://localhost:1337/
+VUE_APP_MODE=d
```

Comment:
what is it ?

Reply:
![image](https://user-images.githubusercontent.com/3717296/77066016-a598c780-69e2-11ea-9678-97fb01987666.png)
Its useful to know if extension if dev or prod version